connectors/src/connectors/intercom/temporal/config.ts

```diff
@@ -1,2 +1,2 @@
-export const WORKFLOW_VERSION = 1;
+export const WORKFLOW_VERSION = 2;
```

Comment:
Be careful as any outstanding workflow won't be picked up by workers anymore as you deploy

Reply:
Given that we have long lived cron workflows that means you have to restart them all right?

Reply:
Yes I will restart them all!